packages/transition-frontend/webpack.config.js

```diff
@@ -176,8 +179,10 @@ module.exports = (env) => {
       )
     ],
     resolve: {
+      mainFields: ['browser', 'main', 'module'],
```

Comment:
I forgot to ask about the changes here, but at this point, I guess it's ok.

Reply:
I guess anyone who knows webpack better than us can figure it out...

I'll open an issue to revisit webpack config file anyway, I'm sure there's tons of optimization to make